tests/utils.py

```diff
@@ -110,8 +110,8 @@ def create_empty_settings(test_type: str) -> ModuleNamespace:
 
 class BaseVcsClient:
     def __init__(self):
-        self.root_directory: py.path.local
-        self.repo_file: py.path.local
+        self.root_directory: py.path.local  # type: ignore
```

Comment:
In my opinion it would be much better to annotate the function than to suppress warning. The warning is valid.